docs/2/admin/admin-management/user-management

```diff
@@ -0,0 +1,115 @@
+
+ 
+### Remove Existing Users  
+--------------------------------------- 
+
+If you are the administrator of an identity you will be able to remove people from that identity as needed. To better understand the identities you can read our [identity guide.](../../../technical/microshare-platform-advanced/identity-guide/ 
+
+**1.** To determine user access, first log into Microshare.  
+ 
+<br> 
+**2.** Head over to the Manage tab. From there click on the Keys tab. 
+ 
+<br> 
+{% include image.html url="/assets/img/admin/identity_0.png" description="Banner" %} 
+ 
+<br> 
+**3.** Then go under the Organization tab.  
+ 
+<br> 
+{% include image.html url="/assets/img/admin/identity_1.png" description="Banner" %} 
+ 
+<br> 
+**4.** If you are the administrator of an identity the users button will appear next to it. As you can see above, the account is an administrator of the `External Identity` and not of `Microshare Default`. Click on users next to the identity that you want to remove a user from. 
+
+<br> 
+**5.** Locate the user you would like to remove from the identity from the users list displayed. Click the Remove button to right of the desired user.  
+
+<br> 
+{% include image.html url="/assets/img/admin/identity_2.png" description="Banner" %} 
+
+<br> 
+**6.** Repeat for the other users as necessary. 
+
+ 
+
+ 
+
+### Disable/Enable Existing User Accounts  
+--------------------------------------- 
+
+If you are the administrator of an identity, you will be able to disable or enable user accounts in that identity as needed. 
+
+**1.** Follow steps 1 through 4 from the Removing the Existing Users section above. 
+
+<br> 
+**2.** Locate the user you would like to disable or enable from the users list displayed. If a user is currently enabled, a Disable button will be present next to their name. If a user is currently disabled, an Enable button will be present next to their name. Depending on the scenario, click either the Disable or Enable button to right of the desired user. 
+
+<br> 
+{% include image.html url=" assets/img/admin/Managing Users 1 Enable-Disable User.jpg" description="Banner" %} 
+
+<br> 
+**3.** Repeat for the other users as necessary. 
+
+ 
+
+ 
+
+### Assign User Role  
+--------------------------------------- 
+
+If you are the administrator of an identity, you will be able to assign roles to users in that identity as needed. 
+
+**1.** Follow steps 1 through 4 from the Removing the Existing Users section above. 
+
+<br> 
+**2.** Locate the user you would like to assign a role to. In the Roles column, type the desired role into the text box, next to the desired user. Then, click the update button directly underneath the text box.  
+
+<br> 
+{% include image.html url=" /assets/img/admin/Managing_Users_Roles1.jpg" description="Banner" %} 
```

Comment:
maybe specify that for now this is not really used ? but this will have an impact in the future ?